src/pages/news/news-add/news-add.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { useStyles } from './news-add.styles';
+import NewsForm from '../news-form/news-form';
+
+const NewsAdd = () => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.container}>
```

Comment:
So, if you named this class container it would be better to move it to containers folder.